patterns/add-homomorphic/README.md

```diff
@@ -0,0 +1,14 @@
+# Additive homomorphic encryption
```

Comment:
Some things I would change here:
- I would add the links to all mentioned articles/repos
- I would add the most basic explanation of why AHE is useful, like the usual `Enc(a) + Enc(b) = Enc(a+b)`.
- I would also note that a reason to use it is to avoid creating a bunch of notes due to the use of public state, but at the cost of leaking that a tx updated the balance, and if that balance were to be tied to anyone, it would leak whose balance was updated as well.